crates/dbs-virtio-devices/src/notifier.rs

```diff
@@ -0,0 +1,248 @@
+// Copyright 2019 Alibaba Cloud. All rights reserved.
```

Comment:
I would be better to move this into dbs-interrupt

Reply:
Issue #69 has been opened to track this.

But I feel that VirtioNotifier is related to Virtio, and it makes sense to put it here.

Reply:
Just move the trait `VirtioNotifier` definition?

Does the specific type `VirtioNotifierLegacy` and `VirtioNotifierMsi` not change?

Reply:
It would be better to move them all and rename them with generic names.

Reply:
fixed